app/javascript/components/games/Rating.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Star from "./star";
+
+const Rating = ({ rating }) => {
+  return (
+    <span className="star-rating">
+      {[1, 2, 3, 4, 5].map(star => (
+        <Star key={star} full={star <= rating || 0} />
+      ))}
+    </span>
+  );
+};
+
+Rating.propTypes = {
+  rating: PropTypes.number || null
```

Comment:
`|| null` is wrong here. `PropTypes.number` already means that it is optional. Use `PropTypes.number.isRequired` in other case

Reply:
Changed it to `PropTypes.number.isRequired` and added a default value of __0__ for `rating` column in database.